include/aws/http/private/h2_frames.h

```diff
@@ -223,7 +223,8 @@ int aws_h2_encode_data_frame(
     size_t *connection_window_size_peer,
     struct aws_byte_buf *output,
     bool *body_complete,
-    bool *body_stalled);
+    bool *body_stalled,
+    bool *body_error);
```

Comment:
utterly trivial: `error` isn't this most boolean-ish word
```suggestion
    bool *body_failed);
```